packages/components/pages/f-account-info/src/components/_tests/AccountInfo.test.js

```diff
@@ -154,20 +154,39 @@ describe('AccountInfo', () => {
         });
     });
 
-    describe('`methods`', () => {
```

Comment:
Personally i think this describe makes it easier to keep things (method tests in this case) together, otherwise it gets hard to find places to add new tests.